### PR TITLE
Adopts the new top nav from the website

### LINKED ISSUE #945: [DOC] Update navigation on Documentation page
**What do you want to do?**
 
- [ ] Request a change to existing documentation
- [ ] Add new documentation
- [x ] Report a technical problem with the documentation
- [ ] Other 

**Tell us about your request.** Provide a summary of the request and all versions that are affected.
 The navigation on the documentation page needs to match the navigation on the rest of the site. 

**What other resources are available?** Provide links to related issues, POCs, steps for testing, etc.
Here is the Doc page: https://opensearch.org/docs/latest/ 
Screen Shot 2022-08-22 at 3 44 41 PM

Here is an example from the Downloads page: 
https://opensearch.org/downloads.html#opensearch
Screen Shot 2022-08-22 at 3 45 00 PM

You'll find our header file in _includes/header.html

### PR DESCRIPTION
**Pull request by AMoo-Miki**
Signed-off-by: Miki <miki@amazon.com>

### Issues Resolved
Fixes #945 

By submitting this pull request, I confirm that my contribution is made under the terms of the Apache 2.0 license.
For more information on following Developer Certificate of Origin and signing off your commits, please check [here](https://github.com/opensearch-project/OpenSearch/blob/main/CONTRIBUTING.md#developer-certificate-of-origin).

### CODE REVIEW

File: _sass/color_schemes/opensearch.scss [hunk 1]
```diff
@@ -70,3 +70,14 @@ $base-button-color: $grey-lt-300;
...
+$content-width: 740px;
+$toc-width: 232px;
+
+$media-queries: (
+  xs: 320px,
+  sm: 462.5px,
+  md: $content-width,
+  lg: $content-width + $nav-width,
+  xl: $content-width + $nav-width + $toc-width
+);
```
Comment (AMoo-Miki) [inline _sass/color_schemes/opensearch.scss:83]: These shouldn't have been in the `custom.scss` file to start with as that file is included after the mixins for the underlying template are applied.

File: _sass/custom/custom.scss [hunk 2]
```diff
@@ -41,15 +41,6 @@ $content-width: 740px !default;
 $table-border-color: $grey-lt-300;
 $toc-width: 232px !default;
```
Comment (AMoo-Miki) [inline _sass/custom/custom.scss:42]: Not sure why but removing `$toc-width` from here threw errors; will investigate later.

File: _sass/custom/custom.scss [hunk 3]
```diff
@@ -409,11 +400,6 @@ body {
   [role="complementary"] {
     @include sans-serif; // reverse font style on sidebar and secondary areas
   }
-
-  // don't activate mobile styles for larger screens
-  @include mq(md) {
-    min-width: 768px;
-  }
```
Comment (AMoo-Miki) [inline _sass/custom/custom.scss:416]: Didn't match the needs of the website.

File: _sass/custom/custom.scss [hunk 4]
```diff
@@ -923,7 +909,7 @@ main {
   @include clearfix;
 }
 
-@media screen and (max-width: 46.24rem) {
```
Comment (AMoo-Miki) [inline _sass/custom/custom.scss:926]: Simply got rid of the magic number from here.

File: _sass/_nested-nav.scss [hunk 5]
```diff
@@ -0,0 +1,130 @@
+#top {
+  z-index: 9;
+}
+
+@media screen and (min-width: 46.25rem) {
```
Comment (AMoo-Miki) [inline _sass/_nested-nav.scss:5]: Leaving these magic numbers in so this file remains identical to the one on the website for ease of update.

**Review by ashwin-pc — APPROVED** [review]

**Review by Naarcha-AWS — APPROVED** [review]

File: _config.yml [hunk 6]
```diff
@@ -184,3 +184,4 @@ exclude:
   - vendor/gems/
   - vendor/ruby/
   - README.md
+  - .idea
```
Comment (Naarcha-AWS) [inline _config.yml:187]: What does this addition do?

**Pull request merged**